wire-library/wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -513,7 +512,7 @@ public TypeSpec generateType(Type type) {
 
   private TypeSpec generateEnum(EnumType type) {
     NameAllocator nameAllocator = nameAllocators.getUnchecked(type);
-    String value = nameAllocator.get("value");
+    String value = "value";
```

Comment:
This looks like a regression?

Reply:
because we cannot rename it.

```kotlin
  public enum class PhoneType(
    public override val value: Int
  ) : WireEnum {
```

If we rename `value`, it won't be overriding anything and the generated code won't compile.

Reply:
It might actually be ok for Java but not Kotlin. I'll confirm it works fine or not on Java before merging

Reply:
And I'll leave a comment on the Kotlin code

Reply:
I don't need to change Java of course... I bugged.

Reply:
merging on green